docs/openj9_support.md

```diff
@@ -72,26 +71,93 @@ show the minimum build configurations that we have tested to build an OpenJDK bi
+| Linux on IBM Z<sup>&reg;</sup> 64-bit       | Ubuntu 16.04              | gcc 4.4.7                       |                |
 | Windows x86 32-bit                          | Windows Server 2012       | Microsoft Visual Studio 2010 SP1|                |
 | Windows x86 64-bit                          | Windows Server 2012       | Microsoft Visual Studio 2010 SP1|                |
 | AIX<sup>&reg;</sup> POWER BE 64-bit         | AIX 7.1 TL04              | xlc/C++ 13.1.3                  |                |
 
-**NOTE:** The level of gcc supported is currently v4.4.7, but plans are in place to move to at least v4.8 in the near future.
+**NOTE:** The level of gcc supported is currently version 4.4.7, but plans are in place to move to at least version 4.8 in the near future.
+
+
+### OpenJDK 9
+
+OpenJDK9 binaries are supported on the minimum operating system levels shown in the following tables:
+
+
+| Linux                                 |  x32   |  x64   |   ppc64le  | Z31  | Z64  |
+|---------------------------------------|--------|--------|------------|------|------|
+| Centos 6                              |   Y    |   Y    |     Y      |   N  |  N   |
+| Centos 7                              |   Y    |   Y    |     Y      |   N  |  N   |
+| Red Hat Enterprise Linux (RHEL) 6     |   Y    |   Y    |     Y      |   Y  |  Y   |
+| RHEL 7                                |   Y    |   Y    |     Y      |   Y  |  Y   |
+| SUSE Linux Enterprise Server (SLES) 12|   Y    |   Y    |     Y      |   Y  |  Y   |
+| Ubuntu 16.04                          |   Y    |   Y    |     Y      |   N  |  Y   |
+| Ubuntu 18.04                          |   Y    |   Y    |     Y      |   N  |  Y   |
 
-OpenJDK8 binaries are supported on the minimum operating system levels shown in the table.
-For Linux, other distributions are supported provided that they have a Linux kernel v3.2 or later
-(Look in the `/proc/version` file to find the kernel version number).
+<i class="fa fa-pencil-square-o" aria-hidden="true"></i><span class="sr-only">Note</span> **Note:** Not all of these distributions are tested, but Linux distributions that have a
+minimum glibc version 2.12 are expected to function without problems.
+
+| Windows                    |  x32   |  x64  |
+|----------------------------|--------|-------|
+| Windows 10                 |   Y    |   Y   |
+| Windows Server 2012        |   N    |   Y   |
+| Windows Server 2012 R2     |   N    |   Y   |
+| Windows Server 2016        |   N    |   Y   |
+
+
+
+| AIX          |  ppc32   | ppc64    |
+|--------------|----------|----------|
+| AIX 7.1 TL4  |    Y     |    Y     |
+| AIX 7.2      |    Y     |    Y     |
 
 When public support for an operating system version ends, OpenJ9 can no longer be supported on that level.
 
-### OpenJDK 9 build environment
+#### OpenJDK 9 build environment
+
+The following table shows the operating system and compilers that the project use to build and test OpenJ9 for different platforms:
```

Comment:
This Linux x86 line in the table shows the kernel level in brackets, which I don't think is required.

Reply:
Fixed